from unittest import TestCase


class TestDebug(TestCase):
  def test_log(self):
    self.fail()